{{cookiecutter.project_slug}}/compose/production/traefik/traefik.toml

```
@@ -0,0 +1,44 @@
+logLevel = "ERROR"
```

I was a bit surprised to not see any logs from the Traefik container, until I saw this. From a simple test, it looks like `INFO` shows some activity without being too noisy. Should we use that instead?

```suggestion
logLevel = "INFO"
```